dot/sync/message_test.go

```diff
@@ -27,7 +27,7 @@ func TestService_CreateBlockResponse(t *testing.T) {
 		want              *network.BlockResponseMessage
 		err               error
 	}{
-		"invalid block request": {
+		"invalid_block_request": {
```

Comment:
why change test case names like this?

Reply:
> Is that some new convention that seems better than what we have already?

Yeah it is, since logs written by `go test` replaces spaces with underscores. So it's easier to copy paste and search in the codebase. We should use underscores instead of spaces from now on (feel free to harass me on that, I'm having trouble remembering this 😄)

Reply:
Yeah, if you have the following test case name: `invalid block request` then the go test output will be `invalid_block_request`, so will be more convenient to introduce the underscore by default while naming a test case.